django/db/models/aggregates.py

```diff
@@ -14,9 +14,9 @@ class Aggregate(Func):
     contains_aggregate = True
     name = None
 
-    def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize=False):
+    def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize=False, for_save=False):
         assert len(self.source_expressions) == 1
-        c = super(Aggregate, self).resolve_expression(query, allow_joins, reuse, summarize)
+        c = super(Aggregate, self).resolve_expression(query, allow_joins, reuse, summarize, False)
```

Comment:
It would be helpful to have a comment here explaining why the received value of `for_save` is discarded instead of passed on to `super` like the other arguments.